manual/winamp/README.md

```diff
@@ -0,0 +1,20 @@
+# <img src="" width="48" height="48"/> [winamp](https://chocolatey.org/packages/winamp)
```

Comment:
Since this is a manual package, the `Update-IconUrl` script is never run, you'll need to add the icon source yourself